packages/experimental/Menu/src/Menu/useMenu.ts

```diff
@@ -3,10 +3,12 @@ import { MenuProps, MenuState } from './Menu.types';
 
 export const useMenu = (props: MenuProps): MenuState => {
   const [open, setOpen] = useMenuOpenState(props);
+  const triggerRef = React.useRef(null);
```

Comment:
quick check -- does changing the target at runtime cause the appropriate re-render/move?

Reply:
Not sure, but I think it should? It's kind of hard to test without coding this out more since right now target is hardcoded just to the trigger that MenuTrigger wraps. Since this is a prototype just to make sure that we can build out the structure we want, I didn't add more functionality than necessary to prove that target will work.

Reply:
no problem! I'm not specifically concerned, it's something of a "technically correct" scenario for menus because there isn't a reason to be moving around vs closing.  I've tried to mind ref runtime behaviors since they aren't state on their own (useRef) and don't cause re-renders which has caught me a couple times.